homeassistant/components/sharkiq/vacuum.py

```diff
@@ -191,6 +197,28 @@ async def async_locate(self, **kwargs: Any) -> None:
         """Cause the device to generate a loud chirp."""
         await self.sharkiq.async_find_device()
 
+    async def async_clean_room(self, rooms: list[str], **kwargs: Any) -> None:
+        """Clean specific rooms."""
+        if len(rooms) == 0:
+            raise exceptions.HomeAssistantError("No rooms to clean were provided.")
+        rooms_to_clean = []
+        valid_rooms = []
+        if self.available_rooms is not None:
+            valid_rooms = self.available_rooms
+        for room in rooms:
+            if room in valid_rooms:
+                rooms_to_clean.append(room)
+            else:
+                LOGGER.error("Invalid room %s", room)
+                raise exceptions.HomeAssistantError(
+                    "One or more of the rooms listed are unavailable to your vacuum. "
+                    "Make sure all rooms match the Shark App, including capitalization."
+                )
+
+        LOGGER.info("Cleaning room(s): %s", rooms_to_clean)
```

Comment:
```suggestion
        LOGGER.debug("Cleaning room(s): %s", rooms_to_clean)
```

Level info is reserved for the core